### PR TITLE
Beta doc updates, remove coverage, fix cli add edit sshkeyfile check. Closes #664 #654 #666.

### LINKED ISSUE #664: Beta doc updates for SELinux and simpler usability
## Specify type:
- Enhancement

## Description:
Capture some of the updates from Elijah on SELinux and usability.

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: docs/source/beta.rst [hunk 1]
```diff
@@ -174,29 +181,34 @@ There are several configurable options that must be considered:
...
+  # mkdir log
 
-  docker run --name quipucords -d -p443:443 -v /opt/quipucords/sshkeys:/sshkeys -v /opt/quipucords/data:/var/data -v /opt/quipucords/log:/var/log -i quipucords:latest
+Following these steps we can now launch the Quipucords server with the following docker command, if your system does not have **SELinux** enabled::
+
+  # sudo docker run --name quipucords -d -p 443:443 -v ~/quipucords/sshkeys:/sshkeys -v ~/quipucords/data:/var/data -v ~/quipucords/log:/var/log -i quipucords:latest
+
+If your system does have **SELinux** enabled you will need to append ":z" to each volume mapping as follows::
+
+  # sudo docker run --name quipucords -d -p 443:443 -v ~/quipucords/sshkeys:/sshkeys:z -v ~/quipucords/data:/var/data:z -v ~/quipucords/log:/var/log:z -i quipucords:latest
```
Comment (kdelee) [inline docs/source/beta.rst]: 👍

File: quipucords/quipucords/settings.py [hunk 2]
```diff
@@ -73,9 +74,12 @@
     'django_filters',
     'drf_generators',
     'api',
-    'coverage'
 ]
 
+if not PRODUCTION:
+    INSTALLED_APPS.append('coverage')
+
```
Comment (kdelee) [inline quipucords/quipucords/settings.py]: 👍

File: quipucords/quipucords/settings.py [hunk 3]
```diff
@@ -56,7 +56,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(os.environ.get('DJANGO_DEBUG', True))
```
Comment (kdelee) [inline quipucords/quipucords/settings.py]: Doesn't have to be this PR but should the default be `False` ?

File: qpc/cred/utils.py [hunk 4]
```diff
@@ -12,30 +12,11 @@
...
-    """
-    keyfile_path = os.path.abspath(os.path.normpath(keyfile))
-    if not os.path.isfile(keyfile_path):
-        print(_(messages.VALIDATE_SSHKEY % (keyfile)))
-        parser.print_help()
-        sys.exit(1)
-    else:
-        # set filename to the resolved keyfile_path
-        return keyfile_path
-
```
Comment (kdelee) [inline qpc/cred/utils.py]: I see you removing the path check for the sshkey in the CLI but don't see you adding anything to the server side. Is that something you want to do later? 🙁 Guess that would make creating a sshkey have to return a `201` and then later report whether or not it worked ¯\_(ツ)_/¯

File: docs/source/beta.rst [hunk 5]
```diff
@@ -174,29 +181,34 @@ There are several configurable options that must be considered:
...
+If you selected to expose port 443 then you would use the following option when running the image ``-p443:443``. If you wish to re-map the port on your system Docker's mapping is -p<host_port>:<container_port>. If you choose for example to re-map the port to 8443 the option to supply would be ``-p8443:443``. Additionally, Docker supplies an option to select a free port for all exposed ports by using the ``-P`` option; the port mapping is then available from the ``sudo docker ps`` command.
 
 
 For the next three configuration options we will take a simple setup strategy for the Quipucords server and create a "home directory" for the server.
 
-1. Create the home directory ``/opt/quipucords``::
+1. Create the home directory ``~/quipucords``::
 
-  mkdir -p /opt/quipucords
+  # mkdir -p ~/quipucords
```
[the remark this answers is not in the document]
Comment (chambridge) [reply docs/source/beta.rst]: I see I was just trying to provide a consistent command prompt symbol. The doc previously had `$`, `#`, or nothing.

File: qpc/cred/utils.py (hunk 4 above)
Comment (chambridge) [reply qpc/cred/utils.py]: Already had this validation correctly on the server:
https://github.com/quipucords/quipucords/blob/master/quipucords/api/credential/serializer.py#L147

Should not have been in the CLI.

File: quipucords/quipucords/settings.py (hunk 3 above)
Comment (chambridge) [reply quipucords/quipucords/settings.py]: Its false in the container environment: https://github.com/quipucords/quipucords/blob/master/Dockerfile#L58

But allows dev to run with it set to True.

**Comment by kdelee** [comment]
FWIW I think maybe this section could get refactored:

```
Verify the server is responding correctly by launching a browser to https://<ip_address>:<port>/admin. If 
your browser is on the same system as the server and you exposed port 443 the URL would be 
https://localhost/admin. When your browser loads you should see the administrative login dialog. From 
here you can log into the server and change the default password. The server comes defaulted with a user 
admin and password pass. You should find the “Change Password” selection in the upper right navigation 
bar.
```

Maybe a heading that says "Verify install" and line break and new sub heading "Change admin password"

File: quipucords/quipucords/settings.py (hunk 3 above)
Comment (kdelee) [reply quipucords/quipucords/settings.py]: ah cool

File: qpc/cred/utils.py (hunk 4 above)
Comment (kdelee) [reply qpc/cred/utils.py]: 🎉 🎈

**Pull request merged**